tests-wtr/dom-actions/dom-actions.element-visible.html

```diff
@@ -0,0 +1,24 @@
+<html>
+<body>
+<script type="module">
+  import {runTests} from '@web/test-runner-mocha';
+
+  runTests(async () => {
+    await import('./dom-actions.element-visible')
+  });
+</script>
+<div id="none-visible">
+    <button style="display:none">This one is invisible</button>
```

Comment:
Could you also add a case when a button element is not hidden itself, but its parent is?
```html
<div style="display:none">
  <button>I am hidden too</button>
</div>
```
There are many more possible cases, but these are probably most common.

Reply:
Do you mean add it as simply as [this](23b3368ddf420404e770c5f484b601e715b46f59)?